doctest/test_docs.py

```diff
@@ -0,0 +1,179 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+import doctest
+import os
+import zc.customdoctests
+import json
+import re
+import random
+import subprocess
+import unittest
+import click
+
+from functools import partial
+from odfe_sql_cli.esconnection import ESConnection
+from odfe_sql_cli.utils import OutputSettings
+from odfe_sql_cli.formatter import Formatter
+from elasticsearch import Elasticsearch, helpers
+
+ENDPOINT = "http://localhost:9200"
```

Comment:
Any chance we can pass this from command line? I'm thinking we may want to use this as sanity test in future.

Reply:
Do you mean we want to connect to an remote endpoint and run doctest? I guess that's not doable for now with `./gradlew doctest`. Currently, `./gradlew doctest` depends on `./gradlew run` which listens at `localhost:9200`. 

However, it may be doable by running this python script only while passing parameters such as the endpoint and login credentials. Need to do some research on that, such as connection with security plugin enabled may require different connection setup with extra arguments passed. Will open an issue to document this

Reply:
Never mind. I forgot that we're dependent on local cluster. We can figure out later if necessary. Thanks!